app/assets/stylesheets/custom.css.scss

```diff
@@ -115,77 +115,125 @@ a:hover, a:active {
 }
 
 #cart:hover {
-    background-color: $ocean-green;
+  background-color: $ocean-green;
 }
 
 #create-button {
-    color: $white;
+  color: $white;
 }
 
 #release {
-    color: $white;
+  color: $white;
 }
 
 .navbar-default {
-    background-color: $parchment;
-    border-color: $parchment;
-    margin: 5px;
-    margin-bottom: 20px;
+  background-color: $parchment;
+  border-color: $parchment;
+  margin: 5px;
+  margin-bottom: 20px;
 }
 
-
+.btn-details {
+  background-color: #ffffff;
+  border-color: $clam-tan;
+  margin-top: 10px;
+  margin: 5px;
+  color: white;
+}
 
 .btn-success {
-    background-color: $dark-ocean-green;
-    border-color: $dark-ocean-green;
-    margin-top: 10px;
-    margin: 5px;
+  background-color: $dark-ocean-green;
+  border-color: $dark-ocean-green;
+  margin-top: 10px;
+  margin: 5px;
 }
 
 .btn-success:hover, .btn-success:focus {
-    background-color: $ocean-green;
-    border-color: $ocean-green;
+  background-color: $ocean-green;
+  border-color: $ocean-green;
 }
 
 .btn-danger {
-    background-color: $light-crab-red;
-    border-color: $light-crab-red;
-    margin-top: 10px;
-    margin: 5px;
+  background-color: $light-crab-red;
+  border-color: $light-crab-red;
+  margin-top: 10px;
+  margin: 5px;
 }
 
 .btn-danger:hover {
-    background-color: $lighter-crab-red;
-    border-color: $lighter-crab-red;
+  background-color: $lighter-crab-red;
+  border-color: $lighter-crab-red;
 }
 
 .btn-success {
-    color: $white;
-    background-color: $dark-ocean-green;
-    padding: 5px;
-    padding-right: 10px;
-    padding-left: 10px;
-    border-color: $dark-ocean-green;
-    margin-top: 10px;
-    margin: 5px;
+  color: $white;
+  background-color: $dark-ocean-green;
+  padding: 5px;
+  padding-right: 10px;
+  padding-left: 10px;
+  border-color: $dark-ocean-green;
+  margin-top: 10px;
+  margin: 5px;
 }
 
 .btn-success:hover {
-    background-color: $ocean-green;
-    border-color: $ocean-green;
+  background-color: $ocean-green;
+  border-color: $ocean-green;
 }
 
 .btn-danger {
-    background-color: $light-crab-red;
-    border-color: $light-crab-red;
-    padding: 5px;
-    padding-right: 10px;
-    padding-left: 10px;
-    margin-top: 10px;
-    margin: 5px;
+  background-color: $light-crab-red;
+  border-color: $light-crab-red;
+  padding: 5px;
+  padding-right: 10px;
+  padding-left: 10px;
+  margin-top: 10px;
+  margin: 5px;
 }
 
 .btn-danger:hover {
-    background-color: $lighter-crab-red;
-    border-color: $lighter-crab-red;
+  background-color: $lighter-crab-red;
+  border-color: $lighter-crab-red;
+}
+
+table {
+ width: 75%;
```

Comment:
Line should be indented 2 spaces, but was indented 1 spaces<br>Properties should be ordered margin-top, width